packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md

```diff
@@ -21,17 +21,18 @@ This page provides the API rate limits for management activities, which is part
+| **Get, update, or delete an application by ID:**<br>`/api/v1/apps/${id}` only                                                      | 100              | 600              | 600     | 600        | 500                   |
+| **Create or list groups:**<br>`/api/v1/groups` except `/api/v1/groups/${id}`                                                       | 100              | 600              | 600     | 600        | 500                   |
+| **Get, update, or delete a group by ID:**<br>`/api/v1/groups/${id}` only                                                           | 100              | 600              | 600     | 600        | 1,000                 |
 | **Create or list users:**<br>Only `GET` or `POST` to `/api/v1/users`                                                               | 100              | 600              | 600     | 600        | 600                   |
-| **Update or delete a user by ID:**<br>Only `POST`, `PUT` or `DELETE` to `/api/v1/users/${id}`                                       | 100              | 600              | 600     | 600        | 600                   |
+| **Update or delete a user by ID:**<br>Only `POST`, `PUT` or `DELETE` to `/api/v1/users/${id}`                                      | 100              | 600              | 600     | 600        | 600                   |
 | **Get System Log data:**<br>`/api/v1/logs`                                                                                         | 20               | 50               | 50      | 50         | 120                   |
 | **Get System Log data:**<br>`/api/v1/events`                                                                                       | 20               | 50               | 50      | 50         | 100                   |
-| **OAuth2 client configuration requests:**<br>`/oauth2/v1/clients`                                                                  | 25               | 100               | 100      | 100        | 100                   |
+| **OAuth2 client configuration requests:**<br>`/oauth2/v1/clients`                                                                  | 25               | 100              | 100     | 100        | 100                   |
 | **Most other API actions:**<br>`/api/v1`                                                                                           | 100              | 600              | 600     | 600        | 1,200                 |
-| **Get a user by ID or sign in**<br>Only `GET` to `/api/v1/users/${idOrLogin}`                                                       | 100              | 1,000              | 1,000     | 1,000      | 2,000                 |
+| **Email Bounces API:**<br>`/api/v1/org/email/bounces/remove-list`                                                                  | 10               | 30               | 30      | 60         | 60                    |
```

Comment:
```suggestion
| **Create an email address bounces remove list:**<br>`/api/v1/org/email/bounces/remove-list`                                                                  | 10               | 30               | 30      | 60         | 60                    |
```
This suggestion tries to be consistent with the other mgmt rate limit listed. Alternatively, another suggestion is:
`**Email Address Bounces requests:**`